changelog/24099.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+auth/ldap: added rotate-root path to ldap auth backend
+```
```

Comment:
The changelog format for features is slightly different to the other types :)

Feel free to reword it

```suggestion
```release-note:feature
**Rotate Root for LDAP auth**: Rotate root operations are now supported for the LDAP auth engine.
```
```